src/lightning/data/streaming/item_loader.py

```diff
@@ -67,6 +67,9 @@ def generate_intervals(self) -> List[Tuple[int, int]]:
     def load_item_from_chunk(self, index: int, chunk_index: int, chunk_filepath: str, begin: int) -> bytes:
         offset = (1 + (index - begin) if index >= begin else index + 1) * 4
 
+        if chunk_filepath in self._chunk_filepaths and not os.path.exists(chunk_filepath):
```

Comment:
if it's meant to be a file, we should use `isfile()`

Reply:
done !